style/README.md

```diff
@@ -33,7 +33,7 @@ Formatting
 * Indent continued lines two spaces.
 * Indent private methods equal to public methods.
 * If you break up a chain of method invocations, keep each method invocation on
-  its own line. Place the `.` at the end of each line, except the last.
+  its own line. Place the `.` at the head of each line.
   [Example][dot guideline example].
```

Comment:
Should probably remove this example while you're here. It doesn't link to anywhere.